src/transformers/models/mt5/modeling_mt5.py

```diff
@@ -31,13 +31,17 @@ class MT5Model(T5Model):
     alongside usage examples.
 
     Examples::
+
         >>> from transformers import MT5Model, T5Tokenizer
         >>> model = MT5Model.from_pretrained("google/mt5-small")
         >>> tokenizer = T5Tokenizer.from_pretrained("google/mt5-small")
         >>> article = "UN Offizier sagt, dass weiter verhandelt werden muss in Syrien."
         >>> summary = "Weiter Verhandlung in Syrien."
-        >>> batch = tokenizer.prepare_seq2seq_batch(src_texts=[article], tgt_texts=[summary], return_tensors="pt")
-        >>> outputs = model(input_ids=batch.input_ids, decoder_input_ids=batch.labels)
+        >>> inputs = tokenizer([article], return_tensors="pt")
+        >>> with tokenizer.as_target_tokenizer():
+        ...     labels = tokenizer([summary], return_tensors="pt")
+
+        >>> outputs = model(input_ids=inputs["input_ids"], decoder_input_ids=labels["input_ids"])
```

Comment:
(nit) I think it would be better to pass `article` and `summary` as either `string` or `list` in all examples to ensure consistency in docs. Some examples are using lists and some are directly passing the single string.